lib/Controller/AttachmentController.php

```diff
@@ -135,7 +135,8 @@ public function uploadAttachment(string $token = ''): DataResponse {
 			return new DataResponse(['error' => 'No uploaded file'], Http::STATUS_BAD_REQUEST);
 		} catch (Exception $e) {
 			$this->logger->error('Upload error', ['exception' => $e]);
-			return new DataResponse(['error' => 'Upload error'], Http::STATUS_BAD_REQUEST);
+			$error = $e->getMessage() ?: 'Upload error';
```

Comment:
Always returning the exception message is not ideal as it may leak internal data to end users, maybe we can limit it to InvalidPathException instances?

Ideally the server would extend HintException and have a error message explicitly specified for the end user, but that is another story I guess 🙈

Reply:
Good catch! I adjusted this now.